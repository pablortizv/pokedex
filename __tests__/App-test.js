import React from 'react';
import {create} from 'react-test-renderer';
import ModalDev from '../src/components/modalDev';


const modal =  create(<ModalDev/>);

test('snapshot', () => {
  expect(modal).toMatchSnapshot();
});
