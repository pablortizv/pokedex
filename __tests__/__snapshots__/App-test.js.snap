// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#56C4C1",
          "flex": 1,
          "marginBottom": 37.5,
          "marginHorizontal": 37.5,
          "marginTop": 170.9,
        },
        Object {
          "backgroundColor": "#56C4C1",
          "borderColor": "#B0B0B0",
          "borderWidth": 10,
          "flex": 1,
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "uri": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/25.png",
        }
      }
      style={
        Object {
          "height": 250,
          "resizeMode": "contain",
        }
      }
    />
    <View
      style={
        Array [
          Object {
            "flex": 1,
          },
          Object {
            "padding": 10,
          },
        ]
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
            "padding": 5,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginTop": 10,
              "paddingHorizontal": 5,
              "width": 65,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View>
            <Text
              style={
                Array [
                  Object {
                    "color": "#225B68",
                  },
                  Object {
                    "textAlign": "center",
                  },
                ]
              }
            >
              React
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginTop": 10,
              "paddingHorizontal": 5,
              "width": 65,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View>
            <Text
              style={
                Array [
                  Object {
                    "color": "#225B68",
                  },
                  Object {
                    "textAlign": "center",
                  },
                ]
              }
            >
              React Native
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginTop": 10,
              "paddingHorizontal": 5,
              "width": 65,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View>
            <Text
              style={
                Array [
                  Object {
                    "color": "#225B68",
                  },
                  Object {
                    "textAlign": "center",
                  },
                ]
              }
            >
              Nodejs
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginTop": 10,
              "paddingHorizontal": 5,
              "width": 65,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#FDFDFD",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View
            style={
              Array [
                Object {
                  "height": 10,
                  "marginBottom": 1,
                  "width": "100%",
                },
                Object {
                  "backgroundColor": "#46A456",
                },
              ]
            }
          />
          <View>
            <Text
              style={
                Array [
                  Object {
                    "color": "#225B68",
                  },
                  Object {
                    "textAlign": "center",
                  },
                ]
              }
            >
              Css
            </Text>
          </View>
        </View>
      </View>
      <Text
        style={
          Object {
            "color": "#225B68",
            "fontSize": 22,
          }
        }
      >
        App created by Pablo Ortiz
      </Text>
      <Text
        style={
          Object {
            "fontSize": 18,
          }
        }
      >
        Mail: pablortizv@gmail.com
      </Text>
      <Text
        style={
          Object {
            "fontSize": 18,
          }
        }
      >
        Phone: +52(Mex) 8116638968 
      </Text>
      <Text
        style={
          Object {
            "fontSize": 18,
          }
        }
      >
        github: github.com/pablortizv
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#22ACFF",
          "borderBottomWidth": 2,
          "borderColor": "#333333",
          "borderRightWidth": 2,
          "bottom": 0,
          "opacity": 1,
          "position": "absolute",
          "right": 0,
          "width": 75,
          "zIndex": 2,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 15,
            },
            Object {
              "textAlign": "center",
            },
          ]
        }
      >
        Close
      </Text>
    </View>
  </View>
</Modal>
`;
